static/js/containers/ProfileImage_test.js

```diff
@@ -0,0 +1,78 @@
+/* global SETTINGS: false */
+import '../global_init';
+
+import React from 'react';
+import { mount } from 'enzyme';
+import { assert } from 'chai';
+import MuiThemeProvider from 'material-ui/styles/MuiThemeProvider';
+import getMuiTheme from 'material-ui/styles/getMuiTheme';
+
+import ProfileImage from './ProfileImage';
+import IntegrationTestHelper from '../util/integration_test_helper';
+
+describe('ProfileImage', () => {
+  let helper;
+
+  const renderProfileImage = (profile, editable=true) => (
+    mount(
+      <MuiThemeProvider muiTheme={getMuiTheme()}>
+        <ProfileImage
+          profile={profile} 
+          store={helper.store}
+          editable={editable}
+        />
+      </MuiThemeProvider>
+    )
+  );
+
+  const thatProfile = {
+    username: 'rfeather',
+    email: 'rf@example.com',
+    first_name: 'Reginald',
+    last_name: 'Feathersworth',
+    preferred_name: 'Reggie'
+  };
+
+  beforeEach(() => {
+    helper = new IntegrationTestHelper();
```

Comment:
Yes, on line 21. I found the `<ProfileImage/>` component throw an error because a `store` wasn't being provided. If there's a better way to accomplish that automatically, I'm all ears because it's not the most elegant to pass something the component should get implicitly like that.